src/main/java/frc/robot/Dashboard.java

```diff
@@ -28,52 +30,82 @@ public class Dashboard extends SubsystemBase {
+                                .getLayout("Drive Tuning", BuiltInLayouts.kList)
+                                .withSize(4, 7);
+
+                slowmodeSpeed = driveTuningLayout
                                 .add("Slowmode Speed", Constants.DrivebaseConstants.DEFAULT_SLOWMODE_SPEED)
                                 .withWidget(BuiltInWidgets.kNumberSlider)
                                 .withProperties(Map.of("min", 0, "max", 1))
                                 .getEntry();
 
-                accelLimitEntry = Shuffleboard.getTab(Constants.ShuffleboardConstants.TUNING_TAB_NAME)
+                accelLimitEntry = driveTuningLayout
                                 .add("Acceleration Limit", Constants.DrivebaseConstants.DEFAULT_MAX_ACCELERATION)
                                 .withWidget(BuiltInWidgets.kNumberSlider)
                                 .withProperties(Map.of("min", 0, "max", 10))
                                 .getEntry();
 
-                deadzone = Shuffleboard.getTab(Constants.ShuffleboardConstants.TUNING_TAB_NAME)
+                deadzoneEntry = driveTuningLayout
                                 .add("Deadzone", Constants.DrivebaseConstants.DEFAULT_DEADZONE)
                                 .withWidget(BuiltInWidgets.kNumberSlider)
                                 .withProperties(Map.of("min", 0, "max", 1))
                                 .getEntry();
 
-                Shuffleboard.getTab(Constants.ShuffleboardConstants.GAME_TAB_NAME)
-                                .add("Drivebase", drivebaseSubsystem.mecanumDrive)
-                                .withWidget(BuiltInWidgets.kMecanumDrive);
+                maxSpeedEntry = driveTuningLayout
+                                .add("Speed Limit", Constants.DrivebaseConstants.DEFAULT_SPEED_LIMIT)
+                                .withWidget(BuiltInWidgets.kNumberSlider)
+                                .withProperties(Map.of("min", 0, "max", 1))
+                                .getEntry();
 
-                Shuffleboard.getTab(Constants.ShuffleboardConstants.GAME_TAB_NAME)
-                                .add("Gyro", drivebaseSubsystem.ahrs)
+                maxRotationSpeedEntry = driveTuningLayout
+                                .add("Rotation Speed Limit", Constants.DrivebaseConstants.DEFAULT_ROTATION_SPEED_LIMIT)
+                                .withWidget(BuiltInWidgets.kNumberSlider)
+                                .withProperties(Map.of("min", 0, "max", 1))
+                                .getEntry();
+
+                telemetryLayout = Shuffleboard.getTab(Constants.ShuffleboardConstants.GAME_TAB_NAME)
+                                .getLayout("Telemetry", BuiltInLayouts.kGrid)
+                                .withProperties(Map.of("Number of columns", 2, "Number of rows", 1))
+                                .withSize(9, 5)
+                                .withPosition(6, 0);
```

Comment:
Most of these positions assume a default size of 64.  This should be communicated in the README or something

Reply:
Will do!